src/test/java/nl/tudelft/ewi/devhub/webtests/AssignmentTest.java

```diff
@@ -28,5 +29,25 @@ public void testListAssignments() {
 		assertEquals(assignment.getName(), "Product vision");
 		assertEquals(assignment.getNumber(), "1");
 	}
+
+	@Test
+	public void testAssignment() throws ParseException {
+		final String net_id = "assistant1";
+		final String password = "assistant1";
+
+		final AssignmentView view = openLoginScreen()
+				.login(net_id, password)
+				.listAssistingCourses()
+				.get(0).click()
+				.listGroups()
+				.get(0).click()
+				.toAssignmentView()
+				.listAssignments()
+				.get(0).click();
+
+		final AssignmentView.Assignment assignment = view.getAssignment();
+        assertEquals("Student Two", assignment.getAuthor());
```

Comment:
We should add some more assertions here, are we looking at the expected assignment?